python/tskit/tables.py

```diff
@@ -3451,6 +3451,44 @@ def union(
             )
 
     def find_ibd(self, samples, max_time=None, min_length=None):
+        """
+        Returns a dictionary containing all IBD (identity-by-descent) segments for the
+        supplied list of sample pairs.
+
+        A pair of nodes ``(u, v)`` has an IBD segment with a left and right coordinate
+        ``[left, right)`` and ancestral node ``a`` iff the most recent common ancestor
+        of the segment ``[left, right)`` in nodes ``u`` and ``v`` is ``a``, and the
+        segment has been inherited along the same genealogical path (ie. it has not
+        been broken by recombination). The segments returned are the longest possible
+        ones.
+
+        Note that this definition is purely genealogical -- allelic states
+        *are not* considered here. If used without time or length thresholds, the
+        segments returned for a given pair will partition the span of the contig
+        represented by the tree sequence.
+
+        :param list samples: A list of pairs of integers, with each pair specified as
+            a tuple. In each pair, the integers correspond to the node IDs of a pair of
+            chromosomes whose IBD segments will be returned by
+            :meth:`.TableCollection.find_ibd`.
```

Comment:
Might as well say "this method" - not much point in linking back to the same place.